python/dgl/graphbolt/impl/ondisk_dataset.py

```diff
@@ -274,10 +278,12 @@ class OnDiskDataset(Dataset):
     def __init__(self, path: str) -> None:
         # Always call the preprocess function first. If already preprocessed,
         # the function will return the original path directly.
+        self.dataset_dir = path
         path = preprocess_ondisk_dataset(path)
         with open(path) as f:
```

Comment:
```suggestion
        yaml_data = preprocess_ondisk_dataset(path)
        with open(yaml_data) as f:
```

Reply:
the `preprocessed_ondisk_dataset` return the `path` of preprocessed metadata.yaml

Reply:
Oh, sorry, I intended to rename it to yaml_path.

Reply:
I'll rename it in next `load` PR～